packages/flutter/lib/src/material/tabs.dart

```diff
@@ -1174,7 +1175,7 @@ class _TabBarViewState extends State<TabBarView> {
     if (widget.controller != oldWidget.controller)
       _updateTabController();
     if (widget.children != oldWidget.children && _warpUnderwayCount == 0)
-      _children = widget.children;
+      _updateChildren();
```

Comment:
this code doesn't make sense. (it already didn't make sense before your change. you're not making it worse.)

It means that while a warp is under way, we stop updating the tab page contents.

Reply:
we stop it here and rely on line 1233 to set it back I presume? It make sense to me although it is not a clean solution

Reply:
The problem is that if, say, there was an animation being driven in the tab, it would suddenly stop happening while you were switching tabs.

We really should fix this.

Reply:
Let's keep that a separate pr. This pr is already too complicated

Reply:
Can you file an issue for changing that? I agree that it should be a separate PR since this one is already complicated.

Reply:
https://github.com/flutter/flutter/issues/31269